### ISSUE TITLE
5.0.3 - Branding - Every change creates new line in branding.ps1 / Config not stored in the DB

### ISSUE THREAD

**Issue by donlent** [ironmansoftware/powershell-universal#3666]
### Version

5.0.1

### Severity

Quality-of-life

### Environment

msi

### Steps to Reproduce

- Setup a Multi-Node Config
- Change the Branding on Node 1
- Perform some changes to the branding
- Again... and Again.... :-)

Check the branding.ps1 file... you'll notice a "history" of changes, where the top most line (1) will be the active branding setting.
![image](https://github.com/user-attachments/assets/c5ede672-53fa-4971-a4c5-136dd9749e71)

In addition the settings should be stored in the DB in my opinion and not per Node / Computer. Like it is on v4...

### Expected behavior

```console
Every change should be stored on all Nodes and in the DB rather than in the branding.ps1.
OR the file has to be "replicated" over to the other node but this is probably more complicated and prone to errors...
```

### Actual behavior

```console
Already mentioned. :-)
```

**Comment by rwidmark** [comment]
It's still important to have the branding.ps1 file as some of us are using git.
And then this is a good approach regarding the branding.ps1

@adamdriscoll

**Comment by adamdriscoll** [comment]
We have an overall plan for optionally storing all configuration files in the database to avoid having to use git at all. This would include the branding file. That said, that won't be included in 5.0 and likely in 5.1 or later due to the size of that type of change. 

In the meantime, we will get this issue with it duplicating records fixed and continue to include it in the git repo.

**Comment by rwidmark** [comment]
> We have an overall plan for optionally storing all configuration files in the database to avoid having to use git at all. This would include the branding file. That said, that won't be included in 5.0 and likely in 5.1 or later due to the size of that type of change. 
> 
> 
> 
> In the meantime, we will get this issue with it duplicating records fixed and continue to include it in the git repo. 

Then I'm guess it's time for me to setup the SQL DB.
It was a to-do that well you know some to-tdos you need a kick in the but to do 🤣
Guessing this is my kick in the but 🤣
